app/pb_kits/playbook/pb_home_address_street/docs/_home_address_street_dark.html.erb

```diff
@@ -1,10 +1,11 @@
 <%= pb_rails("home_address_street", props: {
   address: "70 Prospect Ave",
   address2: "Apt M18",
-  city: "North Arlington",
+  city: "West Chester",
   home_id: 8250263,
   house_style: nil,
```

Comment:
You shouldn't have to pass `nil` to any of the kits

Reply:
I agree with @jasoncypret above. In the documented examples, if we do not want to display home_style, it should be optional and we should not pass it in as a prop.